contrib/epee/src/readline_buffer.cpp

```diff
@@ -192,6 +196,16 @@ static void handle_line(char* line)
   return;
 }
 
+// same_as_last_line returns true, if the last line in the history is
+// equal to test_line.
+static bool same_as_last_line(std::string test_line)
```

Comment:
```suggestion
static bool same_as_last_line(const std::string& test_line)
```

Reply:
Thanks for the feedback! I have also adapted the function declaration and force pushed the change.